assets/css/custom.css

```diff
@@ -0,0 +1,31 @@
+.img-c {
+    width: 100%;
+    height: 18.875rem;
+    object-fit: cover;
+}
+
+.img-xs {
+    width: 100%;
+    height: 10.5rem;
+    object-fit: cover;
+}
+
+.window-100{
+    width: 70%;
+}
+
+.window-80{
+    width: 80%;
+}
+
+.window-70{
```

Comment:
If I remember correctly, these width's were also defined in your component library. You can reuse them instead of re-defining them here

Reply:
you told me that i done